core/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java

```diff
@@ -143,15 +143,17 @@
             Map<String, Settings> groups = s.getAsGroups();
             for (String key : SimilarityService.BUILT_IN.keySet()) {
                 if (groups.containsKey(key)) {
-                    throw new IllegalArgumentException("illegal value for [index.similarity."+ key + "] cannot redefine built-in similarity");
-                }
+                        throw new IllegalArgumentException(
+                                "illegal value for [index.similarity." + key + "] cannot redefine built-in similarity");
+                    }
```

Comment:
Good old Eclipse, failing to notice the world around it. I, also, failed to notice. But I pushed a fix!